packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md

```diff
@@ -18,25 +18,22 @@ An important part of security is to minimize the chances that a malicious actor
 
 **What you need**
 
-* An Okta Developer Edition organization. Don't have one? [Create one for free](https://developer.okta.com/signup).
-* An app that can sign in to Okta. To create your own, see the following guides:
-  * [Sign users in to your web app](/docs/guides/sign-into-web-app/)
-  * [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/)
-  * [Sign users in to your single-page app](/docs/guides/sign-into-spa/)
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* An app that can sign in to Okta
```

Comment:
It seems a bit unhelpful to get rid of the links to the guides where you can create your own sample app to use in this guide. Did you think they were unnecessary, based on the likely skill level of the reader?

Reply:
I moved the same info in the Note to the "What you need" section to avoid repetition.